UI/window-basic-main.cpp

```diff
@@ -805,10 +810,12 @@ static bool LogSceneItem(obs_scene_t*, obs_sceneitem_t *item, void*)
 			? "monitor only"
 			: "monitor and output";
 
-		blog(LOG_INFO, "        - monitoring: %s", type);
+		blog(LOG_INFO, "    %s- monitoring: %s", indent.c_str(), type);
 	}
-
-	obs_source_enum_filters(source, LogFilter, (void*)(intptr_t)2);
+	int child_indent = 1 + (int)(intptr_t)v_val;
```

Comment:
Just a nitpick, might want to change this from `(int)(intptr_t)v_val;` to `val;`
```

Reply:
Also, might want to rename `val` to something like `indent_count` or something.

Reply:
Ah, that makes sense. I named it val to be consistent with LogFilter: https://github.com/obsproject/obs-studio/blob/master/UI/window-basic-main.cpp#L782

I used v_val here like this because I wasn't sure about the best way to do it. I'll fix that up.